account_tax_change/tests/common.py

```diff
@@ -8,8 +8,10 @@
 
 class AccountTaxChangeCommon(AccountTestInvoicingCommon):
     @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(
+            chart_template_ref="l10n_generic_coa.configurable_chart_template"
+        )
```

Comment:
Unless you need a specific COA here you should not override what comes from the method arg.

```suggestion
        chart_template_ref = chart_template_ref or "l10n_generic_coa.configurable_chart_template"
        super().setUpClass(chart_template_ref=chart_template_ref)
```
IF you need it here, otherwise just pass over the argument.